model/ExternalReferenceList.js

```diff
@@ -49,7 +49,7 @@ class ExternalReferenceList {
   }
 
   processExternalReferences (pkg) {
-    if (pkg.homepage) {
+    if (pkg.homepage && pkg.homepage.slice(-1) !== '.') {
```

Comment:
false positives: 
* https://example.com/#seomething.
* https://example.com/?foobar=.
* https://example.com/foo/.
* https://example.com/bar.

Reply:
fixed. tests exist that prove that these cases are no longer false positives